lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -103,6 +104,27 @@ def test_lstchain_mc_rfperformance():
     assert os.path.exists(file_model_energy)
 
 
+@pytest.mark.run(after='test_lstchain_mc_r0_to_dl1')
+def test_lstchain_merge_dl1_hdf5_files():
+    run_program('lstchain_merge_hdf5_files',
```

Comment:
Where does it get it's input files from?

Reply:
merge all hdf5 files in the dir passed with `-d`.
here `output dir`

Reply:
Ah, was confused by the `output_dir`, so here it's actually the input